doc/helpers/responsive-extension.md

```diff
@@ -0,0 +1,109 @@
+---
+uid: Toolkit.UI.ResponsiveExtension
```

Comment:
```suggestion
uid: Toolkit.Helpers.ResponsiveExtension
```

Reply:
The extension is currently in the `Uno.Toolkit.UI` namespace
![image](https://github.com/unoplatform/uno.toolkit.ui/assets/10283398/7d8df8e5-8437-4d66-a2d2-62e8c8cf012e)

Should I move it to the Helpers folder and in the same namespace as the helper?

Reply:
this uid has nothing to do with the class namespace
just how we organize the stuffs in terms how they are structured on the docs site
so far the rules are:
(controls) -> Toolkit.Controls.*
(attached properties, markup extensions, helpers)(or, not controls) -> Toolkit.Helpers.*